catt/controllers.py

```diff
@@ -26,6 +26,20 @@ def get_stream_info(video_url):
     return stream_info
 
 
+def get_chromecast(device_name):
+    devices = pychromecast.get_chromecasts()
+    if device_name:
```

Comment:
Looking at your code again, I see that you are sorting `devices`, regardless of whether it's needed or not, which I do not like. So I ended up reworking my own code, to make it work properly.